source/style-guide/style/placeholder-variable-text.txt

```diff
@@ -4,10 +4,12 @@
 Placeholder (Variable) Text
 ===========================
 
-.. include:: /style-guide/includes/page-needs-update.rst
+.. default-domain:: mongodb
+
+.. include:: /includes/styles/corrections.rst
 
 Placeholder text (also referred to as variable text or replaceable
-text) stands for an object whose specific name is unknown to us.
+text) stands for an object whose specific name we don't know.
```

Comment:
I think we only use placeholders for the values and not the name of variables themselves. There might be some cases I don't know about though. For example, we might use placeholders for:

```
const uri = "<your connection uri>"
```

This type of placeholder probably doesn't need to vary by language. However, we might have variable naming or formatting references to variables and methods such as whether they should contain trailing parens, or use snake/camel case. But from reading the recommendations, I didn't get the impression that was what was being covered. For example, consider the scala "`<variable name>`". I likely would not use a placeholder like this:

```
val <your variable name> = 5;
```

and would instead provide it a descriptive name:

```
val pintsOfBeer = 5;
```

Perhaps the columns in the "Apply Language-Specific Formatting" should only cover what guidelines to use and omit the Format column?

Reply:
This is for the Variable Value. That's a goof on my part.